### DISCUSSION TITLE
Field type: Grid / Table

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6592] [Ideas]
Maybe to advanced for a normal field type... but what about a Grid fielt type (spreadsheets etc).

Check this: http://api.fieldmanager.org/class-Fieldmanager_Grid.html

- **Storage**: serialized
- **Manage**: Allow adding new rows and columns, define column types (head, body, footer)
- **Raw**: Array formatted table with column types and content
- **Display**: Full table (perhaps use DataTables, without the editor it is free)

**Comment by douglsmith** [comment]
The Advanced Custom Fields WordPress plugin has a community add-on to implement a table field type. It seems fairly well done so I'm including a link here for reference. https://wordpress.org/plugins/advanced-custom-fields-table-field/
